src/standalone_pdb/main.cpp

```diff
@@ -8,11 +8,11 @@
 Serial pc(USBTX, USBRX, 9600);  // Serial communication for debugging
 
 // On/off-related inputs/outputs
-DigitalOut buttonLed(LPC_BUTTON_LED, false);     // True means LED on
-DigitalOut mbedLed1(LPC_LED1, false);            // Shows same as button led
-DigitalIn button(LPC_BUTTON_PRESSED, PullDown);  // True means button pressed
-DigitalOut keepPDBOn(LPC_KEEP_PDB_ON, false);    // True means keep PDB on
-DigitalOut mbedLed4(LPC_LED4, false);            // Shows if in Shutdown state
+DigitalOut buttonLed(LPC_ONOFFBUTTON_LED, false);     // True means LED on
+DigitalOut mbedLed1(LPC_LED1, false);                 // Shows same as button led
+DigitalIn button(LPC_ONOFFBUTTON_PRESSED, PullDown);  // True means button pressed
```

Comment:
Maybe you can change this to onOffButton, just like you did for the PDB with EtherCAT. Don't forget to change this throuhout the entire file.

```suggestion
DigitalIn onOffButton(LPC_ONOFFBUTTON_PRESSED, PullDown);  // True means button pressed
```